oobi/benchmark-definitions/python/jax_input_data_definitions.py

```diff
@@ -28,10 +28,51 @@
             "https://storage.googleapis.com/iree-model-artifacts/jax/jax_models_0.4.10_1684396752/RESNET50/batch_${batch_size}/input_0.npy"
         )
     ])
+
+IMAGENET_APPLES_JAX_3X224X224XF16_BATCH_TEMPLATE = data_types_builder.ModelDataTemplate(
+    id=BATCH_ID(unique_ids.INPUT_DATA_IMAGENET_APPLES_JAX_3X224X224XF16),
+    name=BATCH_NAME("IMAGENET_APPLES_JAX_3X224X224XF16"),
+    tags=["input-data", "imagenet", BATCH_TAG],
+    data_format=data_types.DataFormat.NUMPY_NPY,
+    model_id=BATCH_MODEL_ID(unique_ids.MODEL_RESNET50_FP16_JAX_3X224X224XF16),
+    source_info=
+    "Original image: https://storage.googleapis.com/iree-model-artifacts/ILSVRC2012_val_00000023.JPEG",
+    tensor_names=["pixel_values"],
+    tensor_dimensions=[BATCH_TENSOR_DIMS(dims="3x224x224xf16")],
+    source_url=[
+        string.Template(
+            "https://storage.googleapis.com/iree-model-artifacts/jax/jax_models_0.4.10_1684396752/RESNET50/batch_${batch_size}/input_0.npy"
```

Comment:
I wonder if we should create a new directory for each data type? e.g. `RESNET/{fp32,fp16,bf16}/batch{1,...}/` If we decide to regenerate all artifacts then the different datatypes would override each other.

Reply:
Good idea, but I'm not sure we want to do this in the context of this PR as it's fairly large already?
I had a couple of thoughts on organizing the input / output for the benchmarks to simplify the definitions in oobi that I wanted to share, but I haven't gotten around to writing up my thoughts on this.

Reply:
Sounds good. We're in the process of migrating all of this code to `openxla-benchmark` so it's a good time to rethink how to do things.